### DISCUSSION TITLE
SKR Mini E3 V3 with TFT and BBL

### DISCUSSION THREAD

**Discussion by Greyfoxvt** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2721] [General]
I had to write this because I have had so much trouble trying to get the BL Touch to work with this new v3.0 board and the TFT screen on my Creality Ender 3 V2. I have purchased the Bigtree Teck SKR 3.0 v3 with the TFT screen. I also have a BL Touch but have found no documentation or support for firmware for this combination which makes it totally useless. If you are going to promote your great board with touchscreen, you should also support a firmware and instructions tom take this all work. As for no I have reverted back to the Crerality 4.2.7 board and controller. as it at lease they work with BL Touch.

**Comment by bbjay351** [comment]
Check out 3D Printscape on Youtube. He has a tutorial on how to compile firmware and get this particular setup up and running. I have that board with the tft70 and cr touch. Runs great with the Marlin firmware, but now trying to figure out how to compile the professional firmware to work with the tft70. Hope this helps.
